xinference/web/ui/src/scenes/launch_model/modelCard.js

```diff
@@ -124,6 +125,7 @@ const ModelCard = ({ url, modelData, gpuAvailable, is_custom = false }) => {
       quantization: quantization,
       n_gpu:
         nGPU === '0' ? null : nGPU === 'auto' ? 'auto' : parseInt(nGPU, 10),
+      n_gpu_layers: nGPULayers,
```

Comment:
If the user does not choose model_format `ggufv2` / `ggmlv3`, better to not set this parameter. Because these extra parameters would be passed to the backend engine, which may lead to errors.

Reply:
fixed